### PR TITLE
.github/workflows: Fix the podman installation in the e2e tests

### PR DESCRIPTION
**Pull request by timflannagan**
**Description of the change:**

**Motivation for the change:**

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -49,13 +44,8 @@ jobs:
...
       - run: |
-          sudo apt-get update \
-            && sudo apt-get remove msodbcsql17 -y \
-            && sudo apt-get remove buildah podman -y \
-            && sudo apt-get autoremove -y \
-            && sudo apt-get upgrade \
-            && sudo apt-get -y install podman dbus-x11 \
-            && podman version
+          sudo apt-get -y update
+          podman version
```
Comment (timflannagan) [inline .github/workflows/test.yml:48]: Note: whether this was a one-off, but adding the `sudo apt-get -y install podman` here as well resulted in an e2e failure as we attempted to delete a container, which wasn't possible due to `device or resource busy`. Keeping this iteration for now, as we can rely on the podman version (3.0.0) we get by default in the ubuntu LTS release.

**Review by dinhxuanvu — APPROVED** [review]
/approve

**Comment by timflannagan** [comment]
????

```
codecov/project 48.20% (-0.02%) compared to 4078f47
```

**Comment by timflannagan** [comment]
(force pushed with lease so hopefully the codecov/project check stops complaining)

**Pull request merged**